ios/Handlers/EventEmitters/CreatePinEventEmitter.swift

```diff
@@ -0,0 +1,28 @@
+class CreatePinEventEmitter {
+    func onPinOpen(profileId: String, pinLength: UInt) {
+        let data = [
+            "flow": PinFlow.Create.rawValue,
+            "action": PinNotification.open.rawValue,
+            "profileId": profileId,
+            "data": pinLength //TODO: Change this to pinLength
+        ] as [String : Any]
+        BridgeConnector.shared?.sendBridgeEvent(eventName: OneWelcomeBridgeEvents.pinNotification, data: data)
+    }
+    
+    func onPinClose() {
+        let data = [
+            "flow": PinFlow.Create.rawValue,
+            "action": PinNotification.close.rawValue,
+        ]
+        BridgeConnector.shared?.sendBridgeEvent(eventName: OneWelcomeBridgeEvents.pinNotification, data: data)
```

Comment:
don't see OneWelcomeBridgeEvents, but I assume that you can use here and there just `.pinNotification` ?